sentry/src/main/java/io/sentry/CombinedScopeView.java

```diff
@@ -237,6 +237,7 @@ public void setTag(@NotNull String key, @NotNull String value) {
 
   @Override
   public void removeTag(@NotNull String key) {
+    // TODO should this go to all scopes?
```

Comment:
Deleting certain things from scope can be tricky with new API since deletions might go to the wrong scope. Maybe we should just delete from all scopes. On the other hand a user might just want to shadow something temporarily so maybe deleting everywhere isn't wanted.

Reply:
When calling removeX on current scope where scope has been forked since adding e.g. the tag in a parent scope, the remove only removes it from the current scope, not from the parent.

Reply:
Yeah I agree on not removing from all and making it behave the same way `set` does.

Not so sure about setting `null` and returning that because how do you ever unset this?

As a workaround for users actually wanting to clear certain values from all scopes, we could offer a `ScopeType.ALL` where we send `set`, `remove` etc. to all scopes, when they run `Sentry.configureScope(ScopeType.ALL) { scope -> ... }`

Reply:
If users want to shadow something in the sense of temporarily unsetting, we'd have to use `Optional` for all values I guess.